.github/workflows/ci-workflow-custom.yml

```
@@ -0,0 +1,47 @@
+  group: ${{ github.workflow }}-on-${{ github.event_name }}-from-${{ github.ref_name }}
+
+jobs:
+
+  matrix-build:
+    name: Build & dispatch CI matrix
+    uses: ./.github/workflows/ci-matrix-build.yml
+    secrets: inherit
+    with:
+      workflows: nightly
+      matrix_branch: ${{ inputs.matrix_branch }}
+      matrix_file: ${{ inputs.matrix_path }}
```

actionlint (1.7.12)

[error] input "matrix_file" is not defined in "./.github/workflows/ci-matrix-build.yml" reusable workflow. defined inputs are "enable_slack_alert", "matrix_branch", "workflows"

(workflow-call)